next.config.js

```diff
@@ -202,6 +202,16 @@ const nextConfig = {
         destination: 'https://airtable.com/appr1nBRRGx2PTJVh/shrpa99vKKW3xafso',
         permanent: false,
       },
+      {
+        source: '/settings',
```

Comment:
@shelegdmitriy I think it would be sufficient to have only the one redirect which includes `:path*`